flask_jsonapi/query_string.py

```
@@ -114,3 +116,29 @@ def extract_fields(self, value):
 
     def format_resource_paths(self, resource, fields):
         return ['{}.{}'.format(resource, value) for value in fields]
+
+
+class SortParser(QueryStringParser):
+    def __init__(self, schema):
+        self.schema = schema
+
+    def parse(self) -> typing.List[str]:
+        try:
+            sort_fields = flask.request.args.get('sort').split(',')
+            sorting_results = []
+            for sort_field in sort_fields:
+                result = self.get_sorting(sort_field)
+                sorting_results.append(result)
+            return sorting_results
+        except AttributeError:
```

what this except is for?  better not to fail silently

---

Done